src/raftstore/store/peer.rs

```diff
@@ -556,6 +632,30 @@ impl Peer {
                    req: RaftCmdRequest,
                    mut err_resp: RaftCmdResponse)
                    -> bool {
+        if self.merge_state == MergeState::Suspended &&
+           (req.get_requests().len() != 0 || get_split_cmd(&req).is_some() ||
+            get_merge_cmd(&req).is_some() || get_change_peer_cmd(&req).is_some()) {
+            // When in `MergeState::Suspended` state, reject
+            // 1. data read/write requests from client.
+            // 2. region split, region merge, member change commands
+            debug!("{} suspended region rejects data read/write requests and region split, \
```

Comment:
what request do we allow? 
Seem that we can log with "rejects any request execpt xxx"